### PR TITLE
Fix most of the deprecation warnings

### PR DESCRIPTION
**Pull request by FlexW**

### CODE REVIEW

File: src/gui/openfilemanager.cpp [hunk 1]
```diff
@@ -103,27 +103,27 @@ void showInFileManager(const QString &localPath)
...
-            QString nativeArgs;
-            if (!fi.isDir()) {
-                nativeArgs += QLatin1String("/select,");
-            }
-            nativeArgs += QLatin1Char('"');
-            nativeArgs += QDir::toNativeSeparators(fi.canonicalFilePath());
-            nativeArgs += QLatin1Char('"');
-
-            QProcess p;
+            const QString explorer = "explorer.exe "; // FIXME: we trust it's in PATH
```
Comment (mgallien) [inline src/gui/openfilemanager.cpp:106]: indentation looks wrong in github as if there are 4 extra spaces

Comment (FlexW) [reply src/gui/openfilemanager.cpp:106]: I think the problem here is maybe a mix of tabs and spaces.

Comment (mgallien) [reply src/gui/openfilemanager.cpp:106]: no
I did check and I am seeing this when opening the file
image

Comment (mgallien) [reply src/gui/openfilemanager.cpp:106]: `const QString explorer` has 4 extra spaces in from of it that should be removed

Comment (mgallien) [reply src/gui/openfilemanager.cpp:106]: @FlexW ping

Comment (FlexW) [reply src/gui/openfilemanager.cpp]: done

**Review by mgallien — APPROVED** [review]
Cool !!!!

**Comment by mgallien** [comment]
@FlexW a rebase is needed and will hopefully fix tests execution

**Review by allexzander — APPROVED** [review]

**Pull request merged**